intro.js

```diff
@@ -1147,7 +1146,7 @@
   function _mergeOptions(obj1,obj2) {
     var obj3 = {};
     for (var attrname in obj1) { obj3[attrname] = obj1[attrname]; }
-    for (var attrname in obj2) { obj3[attrname] = obj2[attrname]; }
```

Comment:
I'm sorry but I can't get it. Why we shouldn't declare the variable using `var`? And why you think `attrname` is redeclared?